manifests/kustomize/README.md

```diff
@@ -76,6 +76,24 @@ kubectl kustomize base/crds | kubectl delete -f -
 
 ```
 
+## Upgrade
+Note - Do **NOT** follow these instructions if you are upgrading KFP in a
```

Comment:
I'd suggest including "when upgrading from version < xxx, you need to do this". The exact version is still unknown now, but you can put a placeholder there. We will follow up before the release.

Reply:
For the proper kubeflow installation, you mean the [getting-started link](https://www.kubeflow.org/docs/started/getting-started/) instead, right?

Reply:
I pushed the change with the `getting-started link`.
Please confirm that it should be this link.
I can change it if you feel it's better to point to the pipelines page, but then the message should be slightly different, since that link contains information for both types of installation

Reply:
I see, sounds good to me